jobs/config.json

```diff
@@ -1849,7 +1849,7 @@
     ],
     "scenario": "kubernetes_e2e",
     "sigOwners": [
-      "UNKNOWN"
+      "sig-cluster-lifecycle"
```

Comment:
We talked about etcd ownership a bit during the most recent cluster lifecycle meeting and the only consensus was that it didn't have clear ownership. I'm not sure anyone in our sig wants to sign up to own this job atm, but I also don't know who else would own it. It was created by sig-scalability because they were driving the etcd2 --> etcd3 migration. But other than that no sig seems to be owning etcd versioning.